### PR TITLE
Update activities columns

### PR DESCRIPTION
**Pull request by theRoughCode**
Update activities columns:
- [ ] Removed start and end times
- [ ] added is new activity, is open, and state
- [ ] added field validation

### CODE REVIEW

File: src/components/Activities/Activities.jsx [hunk 1]
```diff
@@ -4,114 +4,164 @@ import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
...
+		this.onDeleteRow = this.onDeleteRow.bind(this);
+		this.beforeSaveCell = this.beforeSaveCell.bind(this);
+
+		this.state = {
+			tableData: []
+		};
+	}
+
+	componentDidMount() {
+		//e.preventDefault()
```
Comment (AbhiPrasad) [inline src/components/Activities/Activities.jsx]: probably want to remove this

File: src/components/Activities/Activities.jsx [hunk 2]
```diff
@@ -4,114 +4,164 @@ import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
...
+		var activitiesPromise = getAllActivities();
+		activitiesPromise.then(response => {
```
Comment (AbhiPrasad) [inline src/components/Activities/Activities.jsx]: Do we really need to put this in a variable? Can't we just `.then` the promise right after calling `getAllActivities`?

File: src/components/Activities/Activities.jsx [hunk 3]
```diff
@@ -4,114 +4,164 @@ import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
...
+
+		const newRow = Object.assign({}, row, {
+			id: null,
+			station,
+			grade,
+			isNewActivity,
+			isOpen
+		});
+
+		if (isValid) handleInsertActivities(newRow);
```
Comment (AbhiPrasad) [inline src/components/Activities/Activities.jsx]: 👍

File: src/components/Activities/Activities.jsx [hunk 4]
```diff
@@ -4,114 +4,164 @@ import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
...
+		}
+		if (cellName === 'isNewActivity' || cellName === 'isOpen') {
+			newRow[cellName] = Boolean(cellValue);
+		}
+		handleEditActivities(newRow);
+	}
+
+	render() {
+		if (this.state.tableData.length !== 0) {
+			const inputFieldStyle = {
```
Comment (AbhiPrasad) [inline src/components/Activities/Activities.jsx]: We can probably remove this, it's not being used

**Review by AbhiPrasad — APPROVED** [review]
LGTM

File: src/components/Activities/Activities.jsx (hunk 1 above)
Comment (theRoughCode) [reply src/components/Activities/Activities.jsx]: Oops, thought I did when I merged yours in

File: src/components/Activities/Activities.jsx (hunk 2 above)
Comment (theRoughCode) [reply src/components/Activities/Activities.jsx]: Yea, we don't

**Pull request merged**